File: java/CheckPrimeNumber.java
import java.util.Scanner;

public class CheckPrimeNumber {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a number:");
        int number = sc.nextInt();
        boolean isprime = true;

        if(number>1){
            for( int i=2; i<= Math.sqrt(number); i++){
                if (number%2==0){
                    isprime = false;
                    break;
                }
            }
        }
        if (isprime){
            System.out.println(number + " is a prime number.");
        }
        else{
            System.out.println(number + " is not a prime number.");
        }
        sc.close();
    }
}

File: C/check_prime_number.c
#include<stdio.h>
int main() {
    int number, i, flag = 0;
    printf("Enter a number: ");
    scanf("%d", &number);
    if (number > 1) {
        for (i = 2; i <= number / 2; i++) {
            if (number % i == 0) {
                flag = 1;
                break;
            }
        }
        if (flag == 0)
            printf("%d is a prime number.\n", number);
        else
            printf("%d is not a prime number.\n", number);
    } else {
        printf("%d is not a prime number.\n", number);
    }
    return 0;
}